io.github.efogdev.mpris-timer.yml

```diff
@@ -0,0 +1,43 @@
+id: io.github.efogdev.mpris-timer
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: mpris-timer
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.mpris.MediaPlayer2
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer-0
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer-1
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer-2
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer-3
+  - --talk-name=org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer-4
```

Comment:
this should be automatically granted

> The default policy for the session bus only allows the application to own its own namespace named by $FLATPAK_ID, subnames of it and org.mpris.MediaPlayer2.$FLATPAK_ID for [MPRIS](https://www.freedesktop.org/wiki/Specifications/mpris-spec/).

https://docs.flatpak.org/en/latest/sandbox-permissions.html#d-bus-access

Reply:
`org.mpris.MediaPlayer2.$FLATPAK_ID` should, but the app might be spawned in multiple instances and it is actually expected to. So I'm not sure how else I can handle this
![image](https://github.com/user-attachments/assets/8e8a671f-4117-442f-b9d6-40b7e8a77cd2)

Reply:
make it a sub-bus name of `org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer` instead of adding a `-0` directly eg. `org.mpris.MediaPlayer2.io.github.efogdev.mpris-timer.instance0`